### PR TITLE
DE-240 action_builder dbt cleanup

### PR DESCRIPTION
**Pull request by kanelouise**
Per card [DE-240](https://techallies.atlassian.net/browse/DE-240) and using Jesus's new and improved dbt scripts, these files and folders are named correctly, and the dbt has been tested and run locally. 

Note that the commits are a bit messy because this branch was rebased from main onto `jt/init-dbt-mods`

[DE-240]: https://techallies.atlassian.net/browse/DE-240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .gitignore
Comment (jitoquinto) [inline .gitignore:1]: This probably shouldnt be deleted?

File: dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql [hunk 1]
```diff
@@ -12,8 +11,9 @@ select
     {{ json_extract_scalar('_airbyte_data', ['source'], ['source']) }} as source,
     {{ json_extract_scalar('_airbyte_data', ['status'], ['status']) }} as status,
     {{ json_extract_scalar('_airbyte_data', ['profile'], ['profile']) }} as profile,
-    {{ json_extract_scalar('_airbyte_data', ['entity_id'], ['entity_id']) }} as entity_id,
+    {{ json_extract_scalar('_airbyte_data', ['owner_id'], ['owner_id']) }} as owner_id,
```
Comment (jitoquinto) [inline dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql:14]: hmmm, this seems like it'll remove the `entity_id` column. Are we sure its okay to remove the column?

File: dbt-cta/action_builder/models/sources.yml [hunk 2]
```diff
@@ -1,72 +1,13 @@
 version: 2
 sources:
-- name: cta
+- name: tmc_action_builder
```
Comment (jitoquinto) [inline dbt-cta/action_builder/models/sources.yml:3]: This should stay as `cta`

File: dbt-cta/action_builder/models/sources.yml [hunk 3]
```diff
@@ -1,72 +1,13 @@
...
   database: "{{ env_var('CTA_PROJECT_ID') }}"
   schema: "{{ env_var('CTA_DATASET_ID') }}"
+  quoting:
+    database: true
+    schema: false
+    identifier: false
   tables:
-  - name: _airbyte_raw_action_assignment_group_users
```
Comment (jitoquinto) [inline dbt-cta/action_builder/models/sources.yml:7]: The `_airbyte_raw` tables should stay in the sources

**Review by jitoquinto — CHANGES_REQUESTED** [review]
left a few comments and questions! Also this might have fallen out of sync with the init_dbt branch 🙃

File: .gitignore
Comment (kanelouise) [reply .gitignore:1]: schwoops! yes, thank you. got a lil too yeet-happy.

File: dbt-cta/action_builder/models/sources.yml (hunk 3 above)
Comment (kanelouise) [reply dbt-cta/action_builder/models/sources.yml:7]: Gotcha, thanks!

File: dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql (hunk 1 above)
Comment (kanelouise) [reply dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql:14]: Ahh, I guess I thought that there was probably a schema change where `entity_id` became `owner_id`. That said, I added `entity_id` in. If `entity_id` is empty going forward it won't be an issue, but will just be an empty column, right?

File: dbt-cta/action_builder/models/sources.yml (hunk 2 above)
Comment (kanelouise) [reply dbt-cta/action_builder/models/sources.yml]: Thanks, updated!

**Comment by kanelouise** [comment]
Thanks, @jitoquinto. I rebased this branch to main and made the edits you suggested. Thanks for the eyes!

File: dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql (hunk 1 above)
Comment (jitoquinto) [reply dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql:15]: Yep thats my understanding too! I was looking at the existing tables and it looks like this table is empty so i guess we cant really find out??

Comment (kanelouise) [reply dbt-cta/action_builder/models/0_ctes/social_profiles_ab1.sql:15]: There's less harm in keeping in jic 🤷

**Review by jitoquinto — APPROVED** [review]

**Pull request merged**